src/components/TopNav.tsx

```diff
@@ -117,14 +137,28 @@ export default function TopNav() {
     );
 }
 export const style = {
+    isActive: {
+        backgroundColor: "primary.main",
+        color: "secondary.main",
+    },
     overlay: {
         zIndex: 1210,
     },
+    button: {
+        color: "text.primary",
+    },
     ListItemButton: {
+        color: "text.primary",
         textAlign: "center",
         justifyContent: "center",
         mb: 0.4,
     },
+    divider: {
+        marginLeft: 0.5,
```

Comment:
marginLeft and marginRight can be replaced with mx: 0.5 which is much simpler :)